packages/gatsby/src/redux/machines/page-component.ts

```diff
@@ -124,14 +131,17 @@ module.exports = Machine(
         },
       }),
       deletePage: assign({
-        pages: (ctx, event) => {
+        pages: (ctx, event: AnyEventObject) => {
           ctx.pages.delete(event.page.path)
           return ctx.pages
         },
       }),
       setBootstrapFinished: assign({
-        isInBootstrap: false,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        isInBootstrap: (_ctx, _event): boolean => false,
```

Comment:
We can use
```js
      setBootstrapFinished: assign<IContext>({
        isInBootstrap: false,
      }),
```
to workaround TS problems - according to https://github.com/davidkpiano/xstate/blob/master/docs/guides/context.md#typescript and checked this locally